.travis.yml

```diff
@@ -38,6 +38,10 @@ matrix:
     bundler_args: --without development release
     env: PUPPET_INSTALL_TYPE=agent BEAKER_PUPPET_COLLECTION=puppet6 BEAKER_debug=true BEAKER_setfile=ubuntu1804-64m BEAKER_HYPERVISOR=docker CHECK=beaker
     services: docker
```

Comment:
The task for Puppet 5 is missing.

Reply:
There's no puppet-agent 5.x for focal and I'm unsure if it will be released, but let's wait and see. :)

Reply:
AFAIK it wasn't for Debian 10 so I doubt we'll see it for Ubuntu 20.04

Reply:
There will be no packages for Puppet 5:
* https://tickets.puppetlabs.com/browse/CPR-737